libcloud/utils/networking.py

```
@@ -73,7 +74,10 @@ def is_valid_ip_address(address, family=socket.AF_INET):
     :return: ``bool`` True if the provided address is valid.
     """
     try:
-        socket.inet_pton(family, address)
+        if (platform.system() == 'Windows'):
+            socket.inet_aton(address)
```

In fact there is an issue with IPv6 when using inet_aton.
So we could throw an exception or use an external library dnspython wihich provide inet_aton for ipv4 and ipv6 and for windows also.
Let me know if adding such library is something acceptable or I will just add the throw.